### PR TITLE
Incorrect info fixes (both English & Russian)

### PR DESCRIPTION
**Pull request by Sadykhzadeh**
All contribution in this PR was done with <3 by @AkciumKicum

### CODE REVIEW

**Comment by Sominemo** [comment]
Is this ready for review, or you are still working on it?

**Comment by Sadykhzadeh** [comment]
@Sominemo Ready for review.

File: localization/en/data.json [hunk 1]
```diff
@@ -456,6 +456,11 @@
         "name": "Minimal query length",
         "hint": "for global search",
         "text": "at least 4 characters"
+      },
```
Comment (Sominemo) [inline localization/en/data.json:459]: If you move items between sections, add, or remove them, you need to update `/data/structore.json` too

File: localization/ru-RU/data.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
       "geochats_number": {
         "name": "Количество геочатов",
         "hint": "в которых можно состоять",
-        "text": "максимум 5 (включая созданные вами)"
+        "text": "Неограниченно"
```
Comment (Sominemo) [inline localization/ru-RU/data.json:43]: Any changes to localizations other than English must be submitted though Crowdin only

**Review by Sominemo — CHANGES_REQUESTED** [review]

**Review by Sominemo — APPROVED** [review]

**Pull request merged**